convert2rhel/unit_tests/main_test.py

```diff
@@ -379,6 +379,7 @@ def test_main(monkeypatch):
     toolopts_cli_mock = mock.Mock()
     show_eula_mock = mock.Mock()
     resolve_system_info_mock = mock.Mock()
+    print_data_collection_mock = mock.Mock()
```

Comment:
Let's try to keep the order of the mock consistent for now 

```suggestion
    print_data_collection_mock = mock.Mock()
    resolve_system_info_mock = mock.Mock()
```